applications/nrf_desktop/sample.yaml

```diff
@@ -12,7 +12,7 @@ common:
       - "app_event_manager: e:module_state_event module:main state:READY"
       - "ble_state: Bluetooth initialized"
       - "settings_loader: Settings loaded"
-      - "ble_bond: Selected BLE peers"
```

Comment:
That would not work. We would need to update an application log too. Cannot we use shorter form in application logs?

Reply:
@greg-fer, if you are keen on applying this change, please also align the log in our application module code:

https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf_desktop/src/modules/ble_bond.c#L555
(the first line of the `select_ble_peers` function)

Reply:
@kapi-no , thanks, done.